internal/graph/check_fast_path.go

```diff
@@ -444,6 +444,8 @@ func (c *LocalChecker) resolveFastPath(ctx context.Context, leftChans []chan *it
 		if !leftOpen {
 			return
 		}
+
+		// Capturing a panic here would cause us to wait for the channel to be drained, slowing the overall function down.
 		go drainIteratorChannel(leftChan)
```

Comment:
Adding error handling for panics around these drain functions might cause overall performance to go down since we would have to wait for these channels to drain in order to return an error.